bots/tests-invoke

```diff
@@ -250,6 +243,13 @@ class PullTask(object):
                 os.unlink(dest)
             os.symlink(os.path.join("..", "..", "bots", "naughty", name), dest)
 
+        # COMPAT: Create a legacy tmp/run directory to prevent races during testing
+        try:
+            os.makedirs(os.path.join(BASE, "test", "tmp", "run"))
+        except OSError, ex:
+            if ex.errno == errno.EEXIST:
```

Comment:
I believe you want `!=` here. This is already wrong in the existing code,  but let's fix it while it's being touched.

(Also, all hail to Python 3's  `exist_ok=True` :-) )